test/controllers/observer_controller_test.rb

```diff
@@ -1013,7 +1062,7 @@ def test_create_observation_with_new_herbarium
     assert(herbarium.is_curator?(rolf))
   end
 
-  def test_create_simple_observation_with_approved_unique_name
+  def test_create_simple_observation_with_approved_only_name
```

Comment:
Not sure why you're making this change in this PR, but "approved_only_name" seems awkward to me.  Maybe "approved_single_name"?